linode/rdns/framework_resource.go

```diff
@@ -143,14 +143,20 @@ func (r *Resource) Update(
 	}
 
 	if resourceUpdated {
-		_, err := client.UpdateIPAddress(ctx, plan.Address.ValueString(), updateOpts)
+		ip, err := client.UpdateIPAddress(ctx, plan.Address.ValueString(), updateOpts)
 		if err != nil {
 			resp.Diagnostics.AddError(
 				"Failed to update the Linode RDNS",
 				err.Error(),
 			)
 			return
 		}
+		plan.parseComputedAttributes(ip)
+	} else {
+		// computed attributes without stringplanmodifier.UseStateForUnknown()
+		// will be an unknown value when during an update operation, and it's
+		// required to be a known or null value in the response.
+		plan.ID = plan.Address
```

Comment:
why don't we apply `UseStateForUnknown()` to ID? IMO it should remain no change during the plan update.

Reply:
@yec-akamai You are right.

I didn't apply `UseStateForUnknown()` because I thought the address change can make the ID change. But now I realize that the address change will force a replacement of the resource and the ID will still be shown as `(known after apply)` anyway, so I think we can apply it.